examples/bri-3/src/bri/identity/bpiSubjects/api/dtos/response/bpiSubject.dto.ts

```diff
@@ -1,9 +1,16 @@
-import { AutoMap } from "@automapper/classes";
+import { AutoMap } from '@automapper/classes';
 
 export class BpiSubjectDto {
+  
+  @AutoMap()
```

Comment:
i believe we do not need automap if we use profiles, and vice versa. I am more leaning towards using profiles, as we keep the mapping logic in a single, encapsulated place.

Reply:
AFAIK, profiles picks all the fields marked with @Automap when it triggers the conversion.

Reply:
From my understanding you can either use @automap or use profile with explicitly declared mapping. You do not need both. I am more for profiles for the reason stated above - we do not need to pollute code with @AutoMap decorator.

Reply:
let me try this. I actually didn't try this

Reply:
I see @automap is used everywhere. We can change to profiles at a later date, not to block PR anymore.